docs/03-endpoints/index.md

```diff
@@ -0,0 +1,13 @@
+<!---
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+-->
+
+# DSP-API Endpoints
+
+The endpoints include:
+
+* DSP-API versions 1 and 2, which are intended to be used by clients for querying, creating and updating data
+* DSP-API admin endpoint, which provides admin functionalities. It is intended to to be used only by 
+  [DSP-APP](https://github.com/dasch-swiss/dsp-app) and the DSP itself.
+* DSP-API util routes, which provide information about the DSP stack
```

Comment:
arguably this file could be removed entirely. I don't think it provides much value. Also, normally we use `-` not `*` for lists

Reply:
I removed it.